app/views/mailing_list/steps/completed/shared/_book_a_callback.html.erb

```diff
@@ -1,5 +1,4 @@
-<div class="callback-module">
-  <h2 class="heading-m">Have a question?</h2>
+<h3 class="heading-s">Have a question?</h3>
 
   <p>
```

Comment:
I think that your `<p>` content should now line up with your `<h3>` rather than be indented @emmaachesongray ?